src/utils.ts

```diff
@@ -2,6 +2,17 @@
 
 var addressCodec = require("ripple-address-codec");
 
+/**
+ * A simple property bag which contains components of a classic address. Components contained in this object are neither sanitized or validated.
+ */
+export interface ClassicAddress {
+  /** A classic address. */
+  address: string;
+
+  /** An optional tag. */
+  tag: number | undefined;
```

Comment:
instead of defining it as tag: number | undefined should we instead use the optional parameter syntax of tag?: number?